apps/dialtone-documentation/docs/guides/contributing/index.md

```diff
@@ -40,7 +44,7 @@ If you need to add an icon into Dialtone, here’s how you would go about doing
 5. Open a pull request.
 6. Once approved it can be merged into staging and will go out in the next release.
 
-### For spot illustrations
+### For spot illustrations in Dialtone CSS
```

Comment:
We can remove this entire section now can't we? spot illustrations shouldn't be in dialtone css anymore after this.

Reply:
Huh, I guess it is still necessray, this section is this one that @fede-dp built. https://dialtone.dialpad.com/design/illustrations/

Not sure, if he can confirm it to us would be better.

Reply:
After discussion, we will remove it (https://dialtone.dialpad.com/guides/contributing/#for-spot-illustrations).
It is not necessary anymore.